services/web/server/src/simcore_service_webserver/storage/schemas.py

```diff
@@ -21,12 +22,6 @@ class FileLocationArray(BaseModel):
     __root__: list[FileLocation]
 
 
```

Comment:
not sure why this one goes away here and is added in the other schema?

Reply:
Good question. It's required by the generation of the schema in both the OpenAPI of the webserver and storage. That's why it's shared via the common models.

Reply:
just wondering why this schema then contain some parts. I did not see they were inheritated. ok.